src/responses.ts

```diff
@@ -0,0 +1,2430 @@
+export interface Im {
```

Comment:
could some of these interfaces be unified with interfaces defined in `src/methods.ts`? i know that these define the API outputs, and those the API inputs. unifying them could come with the potential win that similar structures only need to be updated in one place. keeping them separate comes with the win of decoupling. just food for thought at this point.